docs/reference/components/sources/kubernetes_logs.cue

```diff
@@ -202,6 +202,7 @@ components: sources: kubernetes_logs: {
 				syntax: "literal"
 			}
 		}
+		timezone: configuration._timezone
```

Comment:
I realize this setting won't have any effect on most components, but I'm wondering if we shouldn't just have it appear everywhere for consistency, in the future, once all of the components that can support it, do (meaning remap and the syslog source).

Reply:
I don't think it should appear in places where it has no effect. There is a global setting that can serve as a default that fills that role.

Reply:
👍 that's fair.